enkit/version/version.go

```diff
@@ -0,0 +1,42 @@
+package version
+
+import (
+	"fmt"
+
+	"github.com/enfabrica/enkit/lib/client"
+	"github.com/enfabrica/enkit/lib/stamp"
+
+	"github.com/spf13/cobra"
+)
+
+type Root struct {
+	*cobra.Command
+	*client.BaseFlags
+}
+
+func New(base *client.BaseFlags) *Root {
+	rc := &Root{
+		Command: &cobra.Command{
+			Use:           "version",
+			Short:         "Show version info",
+			SilenceUsage:  true,
+			SilenceErrors: true,
+			Long:          "version - command for getting the version of this tool",
```

Comment:
nit - if the long description doesn't expand on what the short description says, i dont think we need to bother.